includes/definitions/aos-emu2.yaml

```diff
@@ -2,9 +2,50 @@ os: aos-emu2
 text: 'APC Environmental Monitoring Unit'
 type: environment
 icon: apc
+over:
+    - {graph: device_temperature, text: Temperature}
+    - {graph: device_humidity, text: Humidity}
 mib_dir:
     - apc
 discovery:
     - sysObjectID:
         - .1.3.6.1.4.1.318.1.3.8.2
         - .1.3.6.1.4.1.318.1.3.8.3
+discovery_modules:
```

Comment:
It is kind of pointless and misleading to disable all these modules.  Most don't do anything on your device.